docs/REST API/rest-api.md

```diff
@@ -9482,6 +9484,8 @@ _Available in Fleet Premium._
 
 Install software (package or App Store app) on a macOS, iOS, iPadOS, Windows, or Linux (Ubuntu) host. Software title must have a `software_package` or `app_store_app` added to be installed.
 
+If the software install takes longer than 1 hour, Fleet will stop the install.
```

Comment:
:+1: that's correct, only thing to keep in mind (worth mentioning?) is that this only applies to software packages (i.e. installs that use scripts), we don't have control on VPP app installation as this is an MDM-controlled command, we just send the command and Apple's MDM protocol takes care of the rest.

Reply:
Ah, thanks Martin. It makes sense to be explicit here.

I updated the language to "custom package" instead of "software"